src/app/tracks-layer/tracks-layer.component.ts

```diff
@@ -41,6 +41,9 @@ export class TracksLayerComponent implements OnInit {
                     2000);
             });
         })
+
+        // this.tracks$ = Observable.from([{entity:Cesium.Cartesian3.fromRadians(Math.random(), Math.random()), actionType:ActionType.ADD_UPDATE, id:1}]);
```

Comment:
I tried to run the code and got cesium length exception!
fix it. 
hit: merge from the master it should resolve it